raw/dune

```diff
@@ -0,0 +1,196 @@
+(library
+ (name raw)
+ (public_name hacl-star-raw)
+ (libraries ctypes ctypes.stubs lib_stubs lib_bindings)
+ (modules)
+ (wrapped false))
+
+(rule
+ (deps configure
+       (source_tree ../kremlin))
+ (targets
+  config.h
+  Makefile.config)
+ (action
+  (no-infer
+    (run ./configure))))
+
+(library
+  (name lib_evercrypt_so)
+  (public_name hacl-star-raw.lib-evercrypt-so)
+  (modules)
+  (foreign_archives evercrypt))
+
+(rule
+  (deps config.h Makefile Makefile.config (source_tree .))
+  (targets libevercrypt.a dllevercrypt.so)
+  (action
+    (no-infer
+      (progn
+        (run make libevercrypt.so libevercrypt.a)
+        (bash "mv libevercrypt.so dllevercrypt.so")))))
+
+(rule
+  (deps lib_gen/Hacl_Spec_gen.exe)
+  (targets Hacl_Spec_stubs.ml Hacl_Spec_c_stubs.c)
+  (action
+    (progn
+      (system "mkdir -p lib")
+      (run %{deps})
```

Comment:
`run` infers that its first argument is a dependency so you can use `(run lib_gen/Hacl_Spec_gen.exe)`, remove it from deps, and it'll do the right thing.

Reply:
(same below)

Reply:
I also applied this on the generated file which made it a good deal shorter :+1: